app/views/DR/UserInfoScreen/index.js

```diff
@@ -119,8 +120,8 @@ export default function UserInfo({ navigation }) {
           let { positive } = await validateCovidPositive(data.body);
           closeDialog(false);
           if (positive) {
-            if (use === 'mySelf') {
-              storeData('positive', positive);
+            if (usage === 'mySelf') {
```

Comment:
```suggestion
            if (use === 'mySelf') {
```

Reply:
Do you want to change all relationate with usage? because is not only in that line